include/grpc++/credentials.h

```diff
@@ -44,9 +44,17 @@ class ChannelArguments;
 class Channel;
 class SecureCredentials;
 
+/// A credentials object encapsulates all the state needed by a client to
+/// authenticate with a server and make various assertions, e.g., about the
+/// client’s identity, role, or whether it is authorized to make a particular
+/// call.
+///
+/// \see https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md
```

Comment:
I think doxygen can suck up these .md files into the generated ball... something we want to do?